posthog/models/error_tracking/error_tracking.py

```diff
@@ -58,3 +58,15 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
         # converting back to list of lists before saving
         self.merged_fingerprints = [list(f) for f in merged_fingerprints]
         self.save()
+
+
+class ErrorTrackingIssueFingerprint(models.Model):
+    team = models.ForeignKey("Team", on_delete=models.CASCADE, db_index=False)
+    issue = models.ForeignKey(ErrorTrackingGroup, on_delete=models.CASCADE)
+    fingerprint = models.CharField(max_length=400)
```

Comment:
Going with a string here. I know fingerprints are arrays of strings right now but I feel like the intention is to move towards a proper hash that will be stored as a string

Reply:
400 is a "magic number" dotted around our codebase whose significance I don't understand.